src/Tools/AnalyzerRunner/AnalyzerRunnerWorkspaceConfigurationService.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace AnalyzerRunner
+{
+    [ExportWorkspaceService(typeof(IWorkspaceConfigurationService), ServiceLayer.Host), Shared]
+    internal sealed class AnalyzerRunnerWorkspaceConfigurationService : IWorkspaceConfigurationService
+    {
+        public WorkspaceConfigurationOptions Options { get; set; }
```

Comment:
mutable here?  it was readonly in `class WorkspaceConfigurationService`

Reply:
Yes, it's initialized here: https://github.com/dotnet/roslyn/pull/59790/files/06917a20d8590b38138091382e49715a1e1456f1#diff-f7279a0428ebff95df34c9364618b64cc2b6999a8c271f773db8dda4da756747R52 based on `_options.UsePersistentStorage`